src/browser/components/molecules/TaskList/index.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react'
+import Layout from '../../molecules/Layout/index'
+import Task from '../Task/index'
+import styles from './index.css';
+const moment = require('moment')
+
+export default class TaskList extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      tasks: []
+    }
+  }
+
+  componentWillMount(){
+    this.loadTasks()
+  }
+
+  loadTasks(){
```

Comment:
Continuing the conversation from the previous comment, this function should only be updating the state, and should not make an API call.